pkg/kubectl/cmd/apply_test.go

```diff
@@ -592,6 +592,81 @@ func TestApplyNonExistObject(t *testing.T) {
 	}
 }
 
+func TestApplyNonExistPatchObject(t *testing.T) {
```

Comment:
It's `non existent`, but anyway the name is misleading, it's about an empty patch, isn't it?